src/constants/index.ts

```diff
@@ -25,64 +25,116 @@ export const EndScene1 = "EndScene1";
-export const FENCE_IMAGE_1 = "./assets/images/fence_v01.png";
-export const AUTUMN_BACKGROUND_1 = "./assets/images/Autumn_bg_v01.jpg";
+export const DEFAULT_BACKGROUND_1 = "./assets/images/bg_v01.webp";
+export const AUTUMN_BACKGROUND_1 = "./assets/images/Autumn_bg_v01.webp";
 export const AUTUMN_HILL_1 = "./assets/images/Autumn_hill_v01.png";
 export const AUTUMN_PILLAR_1 = "./assets/images/Autumn_sign_v01.png";
 export const AUTUMN_SIGN_1 = "./assets/images/Autumn_sign_v01.png";
 export const AUTUMN_FENCE_1 = "./assets/images/Autumn_fence_v01.png";
-export const WINTER_BACKGROUND_1 = "./assets/images/Winter_bg_01.jpg";
+export const WINTER_BACKGROUND_1 = "./assets/images/Winter_bg_01.webp";
 export const WINTER_HILL_1 = "./assets/images/Winter_hill_v01.png";
 export const WINTER_SIGN_1 = "./assets/images/Winter_sign_v01.png";
 export const WINTER_FENCE_1 = "./assets/images/Winter_fence_v01.png";
 export const WINTER_PILLAR_1 = "./assets/images/Winter_sign_v01.png";
-export const LEVEL_SELECTION_BACKGROUND = "./assets/images/map.jpg";
+export const LEVEL_SELECTION_BACKGROUND = "./assets/images/map.webp";
+export const MAP_ICON = "./assets/images/mapIcon.png";
+export const CANCEL_BUTTON = "./assets/images/close_btn.png";
+export const CLOSE_BUTTON = "./assets/images/map_btn.png";
+export const NEXT_BUTTON = "./assets/images/next_btn.png";
+export const NO_BUTTON = "./assets/images/close_btn.png";
+export const PAUSE_BUTTON = "./assets/images/pause_v01.png";
+export const RETRY_BUTTON = "./assets/images/retry_btn.png";
+export const YES_BUTTON = "./assets/images/confirm_btn.png";
+export const SURE_AND_PAUSE_POPUP = "./assets/images/popup_bg_v01.png";
+export const PLAY_BUTTON = "./assets/images/Play_button.png";
+export const PROMPT_TEXT_BG = "./assets/images/promptTextBg.png";
+export const PROMPT_PLAY_BUTTON = "./assets/images/promptPlayButton.png";
+export const LEVEL_INDICATOR = "./assets/images/levels_v01.png";
+export const BAR_EMPTY = "./assets/images/bar_empty_v01.png";
+export const BAR_FULL = "./assets/images/bar_full_v01.png";
+export const TIMER_EMPTY = "./assets/images/timer_empty.png";
+export const ROTATING_CLOCK = "./assets/images/timer.png";
+export const TIMER_FULL = "./assets/images/timer_full.png";
+export const TUTORIAL_HAND = "./assets/images/tutorial_hand.png";
+export const MAP_ICON_SPECIAL =
+  "./assets/images/map_icon_monster_level_v01.png";
+export const MAP_LOCK = "./assets/images/mapLock.png";
+export const STAR = "./assets/images/star.png";
+export const BACK_BUTTON = "./assets/images/back_btn.png";
+export const WIN_BG = "./assets/images/WIN_screen_bg.webp";
+export const PIN_STAR_1 = "./assets/images/pinStar1.png";
+export const PIN_STAR_2 = "./assets/images/pinStar2.png";
+export const PIN_STAR_3 = "./assets/images/pinStar3.png";
+export const CLOUD_6 = "./assets/images/cloud_01.png";
+export const CLOUD_7 = "./assets/images/cloud_02.png";
+export const CLOUD_8 = "./assets/images/cloud_03.png";
```

Comment:
Question why are these not in webp format?

Reply:
I tried. but it become pixelated and its very noticeable. That's why i leave them as is.